mythril/analysis/report.py

```diff
@@ -103,6 +103,18 @@ def as_dict(self):
 
         return issue
 
+    def _add_false_positive_tags(self):
+        """
+        Adds the false positive to description and changes severity to low
+        """
+        self.severity = "Low"
+        self.description_tail += (
+            " This happened in the internal compiler generated code, "
```

Comment:
I don't want to say this is a false positive, rather just something like: "This issue is reported for internal compiler generated code."
wdyt?